plugins/woocommerce/src/Internal/Admin/Notes/WooSubscriptionsNotes.php

```diff
@@ -22,16 +22,7 @@ class WooSubscriptionsNotes {
 	const CONNECTION_NOTE_NAME    = 'wc-admin-wc-helper-connection'; // deprecated.
 	const SUBSCRIPTION_NOTE_NAME  = 'wc-admin-wc-helper-subscription';
 	const NOTIFY_WHEN_DAYS_LEFT   = 60;
-
-	/**
-	 * We want to bubble up expiration notices when they cross certain age
-	 * thresholds. PHP 5.2 doesn't support constant arrays, so we do this.
```

Comment:
This seems an old left-over; we already use const arrays in the codebase.

Reply:
Can confirm – I found at least one in WC Helper. Also, WooCommerce has required min PHP 7.0 since version 4.0.